src/includes/closures/Shared/CachePathUtils.php

```diff
@@ -2,6 +2,39 @@
 namespace WebSharks\ZenCache\Pro;
 
 /*
+* Cache-path suffix frag (regex).
+*
+* @since 15xxxx Enhancing translation support.
+*
+* @param string $regex_suffix_frag Existing regex suffix frag?
+*
+* @return string Cache-path suffix frag (regex).
+*/
+$self->cachePathRegexSuffixFrag = function ($regex_suffix_frag = CACHE_PATH_REGEX_DEFAULT_SUFFIX_FRAG) use ($self) {
+    if ($regex_suffix_frag === CACHE_PATH_REGEX_DEFAULT_SUFFIX_FRAG) {
+        return $self->cachePathRegexDefaultSuffixFrag();
+    }
+    return (string) $regex_suffix_frag;
+};
+
+/*
+* Default cache-path suffix frag (regex).
+*
+* @since 15xxxx Enhancing translation support.
+*
+* @return string Default cache-path suffix frag (regex).
+*/
+$self->cachePathRegexDefaultSuffixFrag = function () use ($self) {
+    if ($self->isPlugin() && !empty($GLOBALS['wp_rewrite'])){
```

Comment:
@jaswsinc I see here that we're only checking if `!empty($GLOBALS['wp_rewrite'])` and then assuming `$GLOBALS['wp_rewrite']->pagination_base` and `$GLOBALS['wp_rewrite']->comments_pagination_base` have valid values. My intuition tells me that might not be the case (i.e., what if `pagination_base` contained a valid value, but `comments_pagination_base` was empty?). 

I haven't reviewed enough of the WP Rewrite code to know for sure, but I wanted to double-check with you before merging this PR.